slices/grep.yaml

```diff
@@ -4,16 +4,16 @@ slices:
   bins:
     essential:
       - libc6_libs
-      - libpcre3_libs
+      - libpcre2-8-0_libs
     contents:
       /bin/grep:
 
   deprecated:
-    # These are shell scripts requiring a symlink from /usr/bin/bash to /bin/sh
+    # These are shell scripts requiring a symlink from /usr/bin/dash to /bin/sh
     # See: https://manpages.ubuntu.com/manpages/jammy/en/man1/grep.1.html
     essential:
-      - bash_bins
-      - libpcre3_libs
+      - dash_bins
```

Comment:
`dash` isn't yet sliced for 22.04 unfortunately

Reply:
https://github.com/canonical/chisel-releases/blob/ubuntu-23.10/slices/dash.yaml includes the symlink that the package installs, it looks like. That make all of the link comments we've been throwing in obsolete, right, if I add a dash slice to `22.04`?

Reply:
Added #126 for Jammy `dash`

Reply:
merged